Free Ruler/de.lproj/MainMenu.strings

```diff
@@ -133,3 +133,6 @@
 
 /* Class = "NSMenuItem"; title = "Inches"; ObjectID = "lt1-Hj-2TR"; */
 "lt1-Hj-2TR.title" = "Inch";
+
+/* Class = "NSMenuItem"; title = "Cycle Units"; ObjectID = "2nm-aL-kZd"; */
+"2nm-aL-kZd.title" = "Einheiten Durchlaufen";
```

Comment:
@J-rg This PR adds a menu item to cycle through the units pixels, millimeters, and inches. Can you review this localized string?

Reply:
The D should not be capitalized.
Though I feel this does not clearly convey what the menu item does; a non-literal translation such as „Nächste Einheit auswählen“ („Select Next Unit“) might be a more intelligible title.

Also, „Inch“ in the line above should be changed to „Zoll“.